code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -625,4 +690,48 @@
 	id = "drunkenblumpkin"
 	description = "A weird mix of whiskey and blumpkin juice."
 	color = "#1EA0FF" // rgb: 102, 67, 0
-	boozepwr = 35
\ No newline at end of file
+	boozepwr = 50
+
+/datum/reagent/consumable/ethanol/whiskey_sour //Requested since we had whiskey cola and soda but not sour.
+	name = "Whiskey Sour"
+	id = "whiskey_sour"
+	description = "Lemon juice/whiskey/sugar mixture. Moderate alcohol content."
+	color = rgb(255, 201, 49)
+	boozepwr = 35
+
+/datum/reagent/consumable/ethanol/fetching_fizz //A reference to one of my favorite games of all time. Pulls nearby ores to the imbiber!
+	name = "Fetching Fizz"
+	id = "fetching_fizz"
+	description = "Whiskey sour/iron/uranium mixture resulting in highly magnetic slurry. Mild alcohol content." //Requires no alcohol to make but has alcohol anyway because ~magic~
+	color = rgb(255, 91, 15)
+	boozepwr = 10
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM
+
+/datum/reagent/consumable/ethanol/fetching_fizz/on_mob_life(mob/living/M)
+	for(var/obj/item/weapon/ore/O in orange(3, M))
+		step_towards(O, get_turf(M))
+	..()
+
+/datum/reagent/consumable/ethanol/hearty_punch //Another reference. Heals those in critical condition extremely quickly. If you die with it in your system, it will be removed and you will live!
+	name = "Hearty Punch"
+	id = "hearty_punch"
+	description = "Brave bull/syndicate bomb/absinthe mixture resulting in an energizing beverage. Mild alcohol content."
+	color = rgb(140, 0, 0)
+	boozepwr = 10
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM
+
+/datum/reagent/consumable/ethanol/hearty_punch/on_mob_life(mob/living/M)
+	if(M.stat == UNCONSCIOUS)
+		M.adjustBruteLoss(-5, 0)
+		M.adjustFireLoss(-5, 0)
+		M.adjustToxLoss(-5, 0)
+		M.adjustOxyLoss(-5, 0)
+		. = 1
+	return ..() || .
```

Comment:
People are going to start using this in cryo mixes in combination with anihol for insanely ridiculous heal speeds. It's not hard to make when all the reagents in this are straight from the dispenser.